standalone/runtime/tests/test_multisig.rs

```diff
@@ -10,12 +10,12 @@ type VestingCall = orml_vesting::Call<Runtime>;
 type UtilityCall = pallet_utility::Call<Runtime>;
 
 #[test]
-fn integration_test_multisig_transfer() {
+fn integration_test_transfer_from_multisig_to_unvested() {
```

Comment:
```suggestion
fn integration_test_transfer_from_multisig_to_vested() {
```

Unvested => needs to vest and are therefore reserved
Vested => free balance as the vesting period for that amount has ended